src/ui/components/NetworkMonitor/index.tsx

```diff
@@ -37,8 +40,22 @@ export const NetworkMonitor = ({
     setTypes(new Set(types));
   };
 
+  let resizeObserver = new ResizeObserver(() => {
+    if (container.current!.offsetWidth > 700) {
+      setVert(true);
+    } else {
+      setVert(false);
+    }
+  });
```

Comment:
smallest nit in the world, but one of my things is unecessary booleans

```js
  let resizeObserver = new ResizeObserver(() => setVert(container.current!.offsetWidth > 700))
```

Reply:
`resizeObserver` should be a ref probably.

Reply:
@abstractalgo can you elaborate on that? I'm not sure what you mean?

Reply:
To create `resizeObserver` via `useRef`, because currently, we're recreating it on every rerender. @jcmorrow

Reply:
ah, thanks, got it!